server/player/scoreboard/scoreboard.go

```diff
@@ -51,9 +51,19 @@ func (board *Scoreboard) Set(index int, s string) (err error) {
 	if index < 0 || len(board.lines) <= index {
 		return fmt.Errorf("index out of range %v", index)
 	}
-
 	// Remove new lines from the string
 	board.lines[index] = strings.TrimSuffix(strings.TrimSuffix(s, "\n"), "\n")
+
+	return nil
+}
+
+// Remove removes a specific line from the scoreboard
```

Comment:
If you add a `.` at the end here this PR is ready to be merged.

Reply:
done